spec/features/visit_spec.rb

```diff
@@ -1,12 +1,50 @@
-require 'spec_helper'
+require 'rails_helper'
 
-paths = %w( / /om /utskott /val /bil /proposals/form /foretag/om /kontakt /logga_in /anvandare/registrera )
+feature 'user visits paths' do
+  let(:user) { create(:user) }
+  let(:album) { create(:album) }
+  let(:cafe_work) { create(:cafe_work) }
 
-feature 'GET' do
-  paths.each do |path|
-    scenario path do
-      visit path
-      page.status_code.should eq(200)
+  paths2 = %w( / /om /utskott /val /bil /proposals/form /foretag/om /kontakt /logga_in /anvandare/registrera )
```

Comment:
Useless assignment to variable - `paths2`. Did you mean `paths`?<br>Line is too long. [110/100]